README.md

```diff
@@ -17,7 +17,9 @@ This includes the following packages:
 * [dapr.conf](./dapr/conf): Configuration
 * [dapr.serializers](./dapr/serializers): serializer/deserializer
 * [dapr.proto](./dapr/proto): Dapr gRPC autogenerated gRPC clients
-* [flask_dapr](./flask_dapr): flask extension for Dapr
+* [dapr.ext.grpc](./ext/dapr-ext-grpc): gRPC extension for Dapr
+* [dapr.ext.fastapi](./ext/dapr-ext-fastapi): FastAPI extension (actor) for Dapr
+* [flask.dapr](./ext/flask-dapr): Flask extension (actor) for Dapr
```

Comment:
is there any reason why flask_dapr is different from the other 2 ext modules? Can they be made uniform?

Reply:
We need to deprecate this one. Flask doesn't support asynchronous method calls correctly but actor is written asynchronously. I saw some race condition issues in hackathon project when it runs in multi-thread scenarios.

User needs to use fastapi extension to use python-actor. I will delete it after this pr.